wall_e/extensions/reaction_role.py

```diff
@@ -33,7 +34,8 @@ def __init__(self):
             )
         self.ROLE_PROMPT = (
             "Time to add roles"
-            "The format to enter roles is emoji then the name of the role or its @, **space delimited**. "
+            "The format to enter roles is emoji then the name of the role or its @, keep them space separated.\n"
```

Comment:
out of curiosity, why did you make a class variable for `CHANNEL_PROMPT`, `TITLE_PROMPT`, `COLOUR_PROMPT` and `ROLE_PROMPT` when they are used only 1 time in one command?

Reply:
left over from an earlier design, some of them were used in edit. I've moved them into the make function now.